### PR TITLE
[UXD] Adding new flags on edit depository IXs

### PR DESCRIPTION
**Pull request by crypto-vincent**
This PR adds the recently added flags:
- mintingDisabled (boolean)
- profitsBeneficiaryCollateral (PublicKey)

to the governance proposal UI

Also removes the CollectProfits IX, since this is a permissionless IX, it does not need to be triggered from the governance

### CODE REVIEW

**Review by acamill — COMMENTED** [review]
Seems fine to me but better wait for some front end guys to review

File: components/inputs/InputNumber.tsx [hunk 1]
```diff
@@ -11,7 +10,7 @@ const InputNumber = ({
   min,
   max,
 }: {
-  value: ReactNode;
+  value: number | undefined;
```
Comment (sjillen) [inline components/inputs/InputNumber.tsx:13]: You sure this doesn't break some other Itxs?

File: pages/dao/[symbol]/proposal/components/instructions/UXD/EditCredixLpDepository.tsx [hunk 2]
```diff
@@ -9,13 +9,17 @@ import Switch from '@components/Switch';
...
 import InputNumber from '@components/inputs/InputNumber';
+import InputText from '@components/inputs/InputText';
 
 const schema = yup.object().shape({
   governedAccount: yup
     .object()
     .nullable()
     .required('Governance account is required'),
   collateralName: yup.string().required('Valid Collateral name is required'),
+  uiRedeemableAmountUnderManagementCap: yup
```
Comment (sjillen) [inline pages/dao/[symbol]/proposal/components/instructions/UXD/EditCredixLpDepository.tsx:20]: Maybe @GregoryNEUT  can confirm here, but I think we try to follow the following convention for any kind of amount:
- use `native` prefix if amount is native
- do not use prefix if amount is UI
So it is a bit less convoluted (smol steps)

**Review by sjillen — APPROVED** [review]
Liking the change on Inputs, but can you confirm it does not break other itxs? 
Otherwise GG

File: components/inputs/InputNumber.tsx (hunk 1 above)
Comment (crypto-vincent) [reply components/inputs/InputNumber.tsx:13]: For sure 👌 looks like other IXs still work fine.

(Tested the RegisterStuff and EditStuff IXs)

that component is always used for the forms like so:

`<InputNumber value={form.blah}/>`

So I think it should be safe

File: pages/dao/[symbol]/proposal/components/instructions/UXD/EditCredixLpDepository.tsx (hunk 2 above)
Comment (crypto-vincent) [reply pages/dao/[symbol]/proposal/components/instructions/UXD/EditCredixLpDepository.tsx:20]: I think the uxd-client names those "uiFields" in some places so I think thats where this comes from. That change is just re-ordering the flags, I didnt actually change any naming here

**Pull request merged**